frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -15,35 +36,34 @@ export const getOCSRequestData = (
     },
     spec: {
       manageNodes: false,
-      storageDeviceSets: [
-        {
-          name: 'ocs-deviceset',
-          count: 1,
-          replica: 3,
-          resources: {},
-          placement: {},
-          portable: true,
-          dataPVCTemplate: {
-            spec: {
-              storageClassName: scName,
-              accessModes: ['ReadWriteOnce'],
-              volumeMode: 'Block',
-              resources: {
-                requests: {
-                  storage,
-                },
-              },
-            },
-          },
-        },
-      ],
+      storageDeviceSets: [createDeviceSet(scName, storage, true)],
```

Comment:
refer this comment - https://github.com/openshift/console/pull/6028#discussion_r457867571